live/macos-config/macos-config.json

```
@@ -165,23 +352,20 @@
         "pproSubscriber": {
           "value": true
         },
-        "pproDaysSinceSubscribed": {
-          "min": 14
-        },
         "pproPurchasePlatform": {
           "value": ["apple", "stripe"]
         },
         "pproSubscriptionStatus": {
-          "value": ["active"]
+          "value": ["expiring", "expired"]
+        },
+        "pproDaysUntilExpiryOrRenewal": {
+          "max": 10
         },
         "appVersion": {
           "min": "1.101.0"
         },
-        "installedMacAppStore": {
-          "value": true
-        },
         "locale": {
-          "value": ["en-US"]
+          "value": ["en-CA", "en-UK"]
```

The valid locale for UK is `en-GB`
```suggestion
          "value": ["en-CA", "en-GB"]
```